R/template-deprecated.R

```diff
@@ -0,0 +1,120 @@
+#' Developer function for summarizing new metrics
+#'
+#'#' @description
+#' `r lifecycle::badge("deprecated")`
+#'
+#' `metric_summarizer()` has been soft-deprecated as of yardstick 1.2.0. Please
+#' switch to use [class_metric_summarizer()], [numeric_metric_summarizer()], or
+#' [prob_metric_summarizer()].
+
+#' @param metric_nm A single character representing the name of the metric to
+#' use in the `tibble` output. This will be modified to include the type
+#' of averaging if appropriate.
+#'
+#' @param metric_fn The vector version of your custom metric function. It
+#' generally takes `truth`, `estimate`, `na_rm`, and any other extra arguments
+#' needed to calculate the metric.
+#'
+#' @param data The data frame with `truth` and `estimate` columns passed
+#' in from the data frame version of your metric function that called
+#' `metric_summarizer()`.
+#'
+#' @param truth The unquoted column name corresponding to the `truth` column.
+#'
+#' @param estimate Generally, the unquoted column name corresponding to
+#' the `estimate` column. For metrics that take multiple columns through `...`
+#' like class probability metrics, this is a result of [dots_to_estimate()].
+#'
+#' @param estimator For numeric metrics, this is left as `NULL` so averaging
+#' is not passed on to the metric function implementation. For classification
+#' metrics, this can either be `NULL` for the default auto-selection of
+#' averaging (`"binary"` or `"macro"`), or a single character to pass along
+#' to the metric implementation describing the kind of averaging to use.
+#'
+#' @param na_rm A `logical` value indicating whether `NA` values should be
+#' stripped before the computation proceeds. The removal is executed in
+#' `metric_vec_template()`.
+#'
+#' @param event_level For numeric metrics, this is left as `NULL` to prevent
+#' it from being passed on to the metric function implementation. For
+#' classification metrics, this can either be `NULL` to use the default
+#' `event_level` value of the `metric_fn` or a single string of either
+#' `"first"` or `"second"` to pass along describing which level should be
+#' considered the "event".
+#'
+#' @param case_weights For metrics supporting case weights, an unquoted
+#' column name corresponding to case weights can be passed here. If not `NULL`,
+#' the case weights will be passed on to `metric_fn` as the named argument
+#' `case_weights`.
+#'
+#' @param ... Currently not used. Metric specific options are passed in
+#' through `metric_fn_options`.
+#'
+#' @param metric_fn_options A named list of metric specific options. These
+#' are spliced into the metric function call using `!!!` from `rlang`. The
+#' default results in nothing being spliced into the call.
+#'
+#' @seealso [metric_vec_template()] [finalize_estimator()] [dots_to_estimate()]
```

Comment:
Can probably remove this

Reply:
done in https://github.com/tidymodels/yardstick/pull/322/commits/9120b16fe04e6dbda19d6b85b88bd4ce44da719a